src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -401,66 +402,41 @@ internal static SqlConnectionAttestationProtocol ConvertToAttestationProtocol(st
         #endregion
 
         #region <<IPAddressPreference Utility>>
-
         /// <summary>
         /// IP Address Preference.
         /// </summary>
-        const string IPAddrPreference46 = "IPv4First";
-        const string IPAddrPreference64 = "IPv6First";
-        const string IPAddrPreferenceOS = "UsePlatformDefault";
-        
+        private readonly static Type s_IPAddressPreferenceType = typeof(SqlConnectionIPAddressPreference);
```

Comment:
You will get better codegen if you use this inline everywhere it's needed instead of using it in a variable.

Reply:
done